.github/workflows/cypress.yml

```diff
@@ -6,7 +6,9 @@ jobs:
     name: Test
     runs-on: ubuntu-18.04
     strategy:
+      fail-fast: false
       matrix:
+        containers: [1, 2]
```

Comment:
Do you plan to use this matrix values anywhere?

Reply:
To my understanding, the containers are being used to run the test in parallel and make load balancing better over time. But I could be wrong. 

https://github.com/cypress-io/github-action#parallel

Reply:
I believe that's correct.  What I think happens is that the matrix means that 2 actions (two 'containers') will fire simultaneously and both will communicate with Cypress's backend.  Because we're passing the `--parallel` flag, the backend will do some load balancing and decide which specs end up in which 'container'.

Reply:
Ah - So the `container` values themselves (1,2, etc.) are "meaningless".

Reply:
It seems pretty weird that it's an array.

Reply:
Yes, because it's part of a matrix of values to create combinations. See: https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstrategymatrix

Reply:
> Ah - So the `container` values themselves (1,2, etc.) are "meaningless".

I think so, yeah.  You might as well use numbers because it looks nice, but I think their only purpose is to create containers.  Could be ['rabbit', 'hat', 'magic'] if we wanted.